WebAPI/appsettings.json

```diff
@@ -8,5 +8,6 @@
       "Microsoft.AspNetCore": "Warning"
     }
   },
+  //"Logging": {"LogLevel": {"": "None"}},
```

Comment:
I can see logging appsettings, but things to think about:
- Boolean to turn logging on/off instead of commenting (commenting is not the best practice);
- There need to be possibility to somehow modify logging as well, so something should be done.

Reply:
It depends what be want to log and how. In my case I use "", to select all logging and not display it. If we want customly do it we would have to specify each case of log and how to manage it.